lib/miq_automation_engine/service_models/miq_ae_service_service_template_transformation_plan_request.rb

```diff
@@ -1,6 +1,7 @@
 module MiqAeMethodService
   class MiqAeServiceServiceTemplateTransformationPlanRequest < MiqAeServiceServiceTemplateProvisionRequest
     expose :source_vms, :association => true
+    expose :validate_conversion_hosts, :association => true
```

Comment:
This is fine but I'm realizing that none of these methods actually return an association so all of the `:associations => true` should be removed.

Even `source_vms` (which should likely be renamed at some point) is only returning IDs not actually objects which is what you would expect for an association, and the naming.

@fdupont-redhat Please remove the `:association = true` for `validate_conversion_hosts` line and I'll create a follow-up PR to resolve the other ones so we do not conflate this PR with both enhancements and code cleanup.

cc @bzwei

Reply:
Thanks @gmcculloug

Reply:
@gmcculloug Done.